source/python/gui/source/__main__.py

```diff
@@ -88,15 +91,21 @@ def causal(args):
     )
 
     if not args.cli:
-        runs = OrderedDict({workload_path: results_df})
+        runs = runs_dict
         kernel_names = ["program1", "program2"]
         max_points = 9
         sortOptions = ["Alphabetical", "Max Speedup", "Min Speedup", "Impact"]
         input_filters = [
             {
                 "Name": "Sort by",
-                "filter": [],
                 "values": list(map(str, sortOptions)),
+                "default": sortOptions[0],
```

Comment:
Why a whole `sortOptions` list if you there is no why for the user to set what they want the default to be and you always index the list at zero? Either add a command-line option and/or a config field or just use `"default": "Impact"` (which should be the default, not alphabetical).

Also, get rid of the "kernel_names" variable. It appears to be left over from some code from OmniPerf bc doesn't appear to be used